TicketBooking/Flights/Services/FlightValidator.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System.Text;
+using TicketBooking.Airports;
+using TicketBooking.Classes;
+using TicketBooking.Countries;
+
+namespace TicketBooking.Flights.Services;
+
+public class FlightValidator
+{
+    private List<Flights.Models.Flight> ExistedFlights { get; init; }
+
+    public FlightValidator(List<Flights.Models.Flight> existedFlights)
+    {
+        ExistedFlights = existedFlights;
+    }
+
+    public string ValidateFlightProperties(
```

Comment:
I believe this function should return a list of strings, and the printer should display the errors.

Reply:
and if you want to make it more muture, it should return array of
```json
{
 "field": <string>,
 "errorMessage": <string>
}
```

But for this task, we are fine with array of string.